flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/mailbox/MailboxImpl.java

```diff
@@ -80,7 +58,7 @@
 	private final Condition notEmpty;
 
 	/**
-	 * Number of letters in the mailbox.
+	 * Number of letters in the mailbox. We track it separately from the queue#size to avoid locking on {@link #hasMail()}.
```

Comment:
nitty nit: Does the `{@link qeuue#size}` work? or `{@link queue}#size`?

Reply:
I tried that, it did not work.